### PR TITLE
Captain Beret can now be attached to Hat Stabilizer MOD

### PR DESCRIPTION
**Pull request by SpaceLoveSs13**
## About The Pull Request

Captain's beret(blue&white) both can now be attached to the Hat Stabilizer Mod.

## How This Contributes To The Skyrat Roleplay Experience

Much like every other hats, berets should also be able to chosen for preference now on top of your hats.

## Changelog
:cl: SpaceLove
add: NT has finally losen up restrictions on Hat Stabilizer Module, allowing captain's beret on top of it as well.
/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
Hi, I added the hat stabilizer upstream. Its intentional. The sprites I chose were specifically chosen because they DONT clip or look ugly on the helmets.

**Comment by SpaceLoveSs13** [comment]
> Hi, I added the hat stabilizer upstream. Its intentional. The sprites I chose were specifically chosen because they DONT clip or look ugly on the helmets.

image
image
image
image

I think they look fine in my opinion and I would rather have the option to choose for the players. Ultimately I will leave the decision upto maintainers.

File: code/modules/mod/modules/modules_general.dm [hunk 1]
```diff
@@ -514,8 +514,8 @@
 			/obj/item/clothing/head/nursehat,
 			/obj/item/clothing/head/chefhat,
 			/obj/item/clothing/head/papersack,
-			)) - /obj/item/clothing/head/caphat/beret
-			//Need to subtract the beret because its annoying
+			/obj/item/clothing/head/caphat/beret
```
Comment (Zonespace27) [inline code/modules/mod/modules/modules_general.dm:517]: you need to make skyrat edit comments for this

**Review by Zonespace27 — CHANGES_REQUESTED** [review]
Take this upstream

File: code/modules/mod/modules/modules_general.dm (hunk 1 above)
Comment (SpaceLoveSs13) [reply code/modules/mod/modules/modules_general.dm:517]: mhm, I will go make a pr there for it

**Pull request closed without merging**